ee/tabby-webserver/src/schema/auth.rs

```diff
@@ -361,23 +362,20 @@ pub trait AuthenticationService: Send + Sync {
     async fn is_admin_initialized(&self) -> Result<bool>;
     async fn get_user_by_email(&self, email: &str) -> Result<User>;
 
-    async fn create_invitation(&self, email: String) -> Result<i32>;
-    async fn list_invitations(&self) -> Result<Vec<Invitation>>;
-    async fn delete_invitation(&self, id: i32) -> Result<i32>;
+    async fn create_invitation(&self, email: String) -> Result<ID>;
+    async fn delete_invitation(&self, id: ID) -> Result<ID>;
```

Comment:
We should always use DTO objects (in this case, schema::InvitationNext / schema::User / ID), instead of DAO objects (db::Invitation / db::User / i32)